XamarinCommunityToolkit/Behaviors/TagBehavior.shared.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class TagBehavior : BaseBehavior<Label>
+    {
+        string RegexPattern => $@"[{string.Join(string.Empty, TagTypes.Select(s => s.Symbol))}]\w+";
```

Comment:
```suggestion
        readonly string RegexPattern => $@"[{string.Join(string.Empty, TagTypes.Select(s => s.Symbol))}]\w+";
```

I think this can be at least `readonly`? Also maybe a little comment explaining what this will match for people not familiar with regex

Reply:
Hey @jfversluis :) That's computed property, so that can't be a read-only.

But I think we could have ```Regex regex;``` filed instead of re-initialize regular expression every time

Reply:
OK, I missed the property part. Next question; why is it a property? :P

Reply:
@jfversluis as I see because it uses another property "TagTypes" that is "dynamic".
But as I said, the best approach for me is storing built Regex.

Reply:
I think it should be read-only, because if I do a built Regex the regex won't be refreshed if TagTypes changes